### PR TITLE
Fix CI workflow errors

### PR DESCRIPTION
**Pull request by roed314**
I've gotten several emails from github describing failed CI runs, and pointing to errors in the workflow files.  This PR attempts to fix these errors.

### CODE REVIEW

**Review by alexjbest — APPROVED** [review]
LGTM

**Review by kwankyu — APPROVED** [review]

**Review by tobiasdiez — CHANGES_REQUESTED** [review]
I think its better to reduce this PR to the fix ofthe cygwin workflow as the other workflows are already fixed in https://github.com/sagemath/sage/pull/34964 and https://github.com/sagemath/sage/pull/34987

File: .github/workflows/ci-cygwin-standard.yml [hunk 1]
```diff
@@ -59,6 +59,7 @@ jobs:
     needs: [cygwin-stage-i-a, cygwin-stage-i-b]
 
   cygwin-stage-ii-e:
+    uses: ./.github/workflows/cygwin.yml
     env:
```
Comment (tobiasdiez) [inline .github/workflows/ci-cygwin-standard.yml:67]: I think the `env` block here also need to be converted into `with`

**Review by tobiasdiez — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-cygwin-standard.yml (hunk 1 above)
Comment (roed314) [reply .github/workflows/ci-cygwin-standard.yml:67]: Thanks, I've updated that.

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
@roed314 the diff here is not empty after merging in develop. What should happen with these changes?

**Comment by roed314** [comment]
I updated the title and description.

**Comment by tobiasdiez** [comment]
I tried to run the updated workflow from the main repo to check that the changes work as intended, but that doesn't work as it is in a different repo.

Can you please activate github actions in your fork, make the `fix-develop` the default branch of your fork, and then manually trigger the cygwin workflow.

**Comment by roed314** [comment]
I manually triggered it, and got the following error:
```
[Invalid workflow file: .github/workflows/ci-cygwin-standard.yml#L67](https://github.com/roed314/sage/actions/runs/4124922889/workflow)
The workflow is not valid. .github/workflows/ci-cygwin-standard.yml (Line: 67, Col: 28): Invalid input, local_artifact_name is not defined in the referenced workflow. .github/workflows/ci-cygwin-standard.yml (Line: 68, Col: 27): Invalid input, logs_artifact_name is not defined in the referenced workflow.
```

I'm working on another issue, but I'll try to come back to this later today.

**Comment by roed314** [comment]
Alright, I don't understand how to write these CI files, and I'm failing to figure out how to fix the problem.  I'll create an issue to deal with this and close this PR (which was originally intended for fixing the checkmarks in CI).

**Pull request closed without merging**